Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7571,8 +7577,8 @@ def initialize_instance(args: Dict[str, str], params: Dict[str, str]):
     # determine a vsys or a device-group
     VSYS = params.get('vsys', '')
 
-    if args and args.get('device-group'):
-        DEVICE_GROUP = args.get('device-group')  # type: ignore[assignment]
+    if args and (device_group := (args.get('device-group') or args.get('device_group'))):
+        DEVICE_GROUP = device_group  # type: ignore[assignment]
     else:
         DEVICE_GROUP = params.get('device_group', None)  # type: ignore[arg-type]
```

Comment:
```suggestion
    DEVICE_GROUP = args and args.get('device-group') or params.get('device_group', None)
```
Replace with constants when you add them in